superflore/generators/bitbake/gen_packages.py

```diff
@@ -35,8 +35,12 @@
 
 
 def regenerate_installer(
-    overlay, pkg, distro, preserve_existing, tar_dir, md5_cache, sha256_cache
+    overlay, pkg, distro, preserve_existing, tar_dir, md5_cache, sha256_cache,
+    skip_keys=None
 ):
+    if skip_keys and pkg in skip_keys:
```

Comment:
Sorry, I didn't realize this until just now... Can we handle this `if` statement in the caller of this function? I think that would make it harder to forget on any platforms that get added in the future.

Reply:
Sure; could you please take a look?